build/Build.cs

```diff
@@ -42,7 +42,7 @@ class BuildNevermore : NukeBuild
         .Executes(() =>
     {
         DotNetRestore(_ => _
-            .SetProjectFile("source"));
+            .SetProjectFile(SourceDirectory));
```

Comment:
The change makes sense to me, do we need feedback from other OS's?

Reply:
It wouldn't hurt to try and build on Mac. I don't think it will be an issue since it works on Linux as we can see [here](https://build.octopushq.com/buildConfiguration/OctopusDeploy_LIbraries_Nevermore?branch=refs%2Fheads%2Fakira%2Ffix-nuke-build&buildTypeTab=overview&mode=builds).

@matt-richardson did you want to try and build this on your Mac?

Reply:
Seems to run fine 👍